pkgs/by-name/jq/jqfmt/package.nix

```
@@ -0,0 +1,29 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  unstableGitUpdater,
+}:
+buildGoModule {
+  pname = "jqfmt";
+  version = "0-unstable-2024-08-15";
+
+  src = fetchFromGitHub {
+    owner = "noperator";
+    repo = "jqfmt";
+    rev = "8fc6f864c295e6bd6b08f36f503b3d809270da61";
+    hash = "sha256-tvFp1SJeosJdCHs3c+vceBfacypJc/aFYSj55mBfkB8=";
+  };
+
+  vendorHash = "sha256-avpZSgQKFZxLmYGj+2Gi+wSDHnAgF0/hyp4HtoQ0ZCo=";
+
+  passthru.updateScript = unstableGitUpdater { };
```

```suggestion
  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch=main" ]; };
```

`unstableGitUpdater` doesn't seem to update the `vendorHash`